godtools/Base.lproj/Localizable.strings

```diff
@@ -1,6 +1,3 @@
-"lessons.filter.navBar.language" = "Lesson language";
```

Comment:
Morning @rachaelblue, looks like these phrases marked for removal were added for the new lessons filter UI.  If you haven't already check with Jill on adding these strings to OneSky.  I'll hold off on merging any automated Pull Requests from the OneSky workflow until these get added in.

Reply:
Thanks @levieggertcru!  Just messaged Jill.